### DISCUSSION TITLE
Planned edits to Documentation

### DISCUSSION THREAD

**Discussion by mzuranski** [Unidata/gempak#122] [General]
- [ ] Incorporate Ubuntu install instructions (https://github.com/Unidata/gempak/issues/57#issuecomment-1066984367).  Note the version number, if nothing else update the version but can I make that dynamic somehow? 🤔 
- [ ] Make notes regarding #76 on the IMCBAR page

Is there anything else worth doing before having Unidata redirect their docs page to here?

**Comment by akrherz** [comment]
Ideally, we would want a github action that built the docs from the GEMPAK main branch and pushed them to the gh-pages branch automagically.

I don't think we need to wait for anything before having Unidata add their redirect.

**Reply by mzuranski** [reply]
> Ideally, we would want a github action that built the docs from the GEMPAK main branch and pushed them to the gh-pages branch automagically.

I'm not sure how to conjure that magic up.  Is there a reason people shouldn't edit the gh-pages branch directly?


> I don't think we need to wait for anything before having Unidata add their redirect.

Awesome, I'll let them know. 👍

**Reply by akrherz** [reply]
> I'm not sure how to conjure that magic up. Is there a reason people shouldn't edit the gh-pages branch directly?

Within the GEMPAK source tree, check out the `docs` folder and all the generation scripts there.  The docs are meant to be built from the source tree and then committed to the `gh-pages` branch via CI.  Reviewing the generation now, there is some work to be done to get there :)
